js-foundations.js

```diff
@@ -3,23 +3,46 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
-
+const divider = (x) => {
+    
+if (x %3=== 0){  
+    return x +" is divisable"
+}else{
+    return x + " is not divisable"
+    }
+}
+console.log(divider(15))
+console.log(divider(0))
+console.log(divider(385))
+console.log(divider(-7))
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
-
+var helloMe = ["Hi my name is Charlie","Montejo","I am a student","that is 21 years old"]
+var helloMeSen = helloMe.join(" ")
+console.log (helloMeSen)
 
```

Comment:
Your output is correct, but the goal of the problem was to create an object called helloMe with key:value pairs of info and a method that returns the info.